src/payment/models/history/dto/history.dto.ts

```diff
@@ -1,3 +1,11 @@
+import { RouteHistoryDto } from '../../route/dto/route-history.dto';
+
+export enum HistoryTransactionType {
+  BUY = 'Buy',
+  SELL = 'Sell',
+  CRYPTO = 'Crypto',
+}
```

Comment:
I would move this to RouteHistoryDto. Maybe we can move the route-history.dto.ts to the history folder?